aerogear/mobile-security-service-operator
# Last Changes before first tag release

## Motivation
https://issues.jboss.org/browse/AEROGEAR-9316

## What
- Allow cluster admin configure image policies (Optional)
- The default value will be empty as requested which will allow the OCP to decide. 
- Remove outdated info from README
- Reorganize CRs file and improve the comments to make clear what is each spec. 
- Add mandatory specs by constants as requested by @wei-lee 
- Do refectories regards comments and instance names (PS.: it is our last chance before since we do not create a tag yet :-))
- Add test to check mandatory specs

**NOTE:** The changes are spit in each commit. You can check the commits' changes to make easier the verification.  

## Why
- In this, the policy can be changed according to the business by its setup/config files. 

## Steps to verify
- Use the image: `docker.io/cmacedo/mobile-security-service-operator:changes `
- Install the operator and check if all still working
- Check the create/delete app and config Map
- Check that the db still using the configMap values for its EnvVars

IMPORTANT: The project is now fully covered by tests and if something brokes is unlike we do not get it so long in the CI. Also, we added a test to ensure the new feat to allowing the usage without specs.
